jsx/src/components/AddUser/AddUser.jsx

```diff
@@ -21,7 +21,7 @@ const AddUser = (props) => {
     });
   };
 
-  var { addUsers, failRegexEvent, updateUsers, history } = props;
```

Comment:
I think this is unused everywhere - can you remove every reference to it, not just this file?

Reply:
OK, no problem. I'll try to find all of them.

Reply:
Hi, @minrk Please kindly review my other PRs, thank you!